### PR TITLE
fix(finish-without-contacts): bug fix from test branch

### PR DESCRIPTION
**Pull request by eladyashar**
solve bug 958
move lastTabDisplaiedId useEffect to investigation form.

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx [hunk 1]
```diff
@@ -1,15 +1,19 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import MuiAlert from '@material-ui/lab/Alert';
 import Snackbar from '@material-ui/core/Snackbar';
 
+import StoreStateType from 'redux/storeStateType';
+import { useSelector } from 'react-redux';
```
Comment (RoeiRom) [inline client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx:6]: Move up

File: client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx [hunk 2]
```diff
@@ -51,9 +58,17 @@ const InvestigationForm: React.FC = (): JSX.Element => {
     const {
         currentTab,
         setNextTab,
-        lastTabDisplayedId
-    } = useTabManagement();
+    } = useTabManagement({lastTabDisplayedId});
+
+    useEffect(() => {
+        console.log('lastTabDisplayedId:' + lastTabDisplayedId);
```
Comment (daniel7764) [inline client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx:64]: do you need this console.log?

Comment (RoeiRom) [inline client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx:64]: Remove console.log

File: client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx (hunk 1 above)
Comment (ItayBenMoshe) [inline client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx:6]: move ot to the first section

**Review by RoeiRom — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx [hunk 3]
```diff
@@ -51,9 +58,17 @@ const InvestigationForm: React.FC = (): JSX.Element => {
...
+    }, [lastTabDisplayedId]);
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx:65]: necessary?

**Review by ItayBenMoshe — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx (hunk 3 above)
Comment (eladyashar) [reply client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx:65]: no, deleted

File: client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx (hunk 2 above)
Comment (eladyashar) [reply client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx:64]: removed

Comment (eladyashar) [reply client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx:64]: removed

File: client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx (hunk 1 above)
Comment (eladyashar) [reply client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx:6]: done

Comment (eladyashar) [reply client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx:6]: done

**Review by daniel7764 — APPROVED** [review]

**Review by RoeiRom — APPROVED** [review]

File: client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx [hunk 4]
```diff
@@ -1,11 +1,10 @@
 import React, { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import StoreStateType from 'redux/storeStateType';
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx:3]: second section

**Review by ItayBenMoshe — APPROVED** [review]

**Pull request merged**